README.md

```diff
@@ -186,6 +187,7 @@ jobs:
 | `build-command`| No       | An optional build commit to run after the version bump and before releasing the package |
 | `api-url`      | No       | GitHub App URL. You wouldn't need to set this unless you're deploying a custom GitHub app instead of [optic-release-automation](https://github.com/apps/optic-release-automation). <br /> (_Default: `https://optic-release-automation-ocrlhra4va-ue.a.run.app/`_) |
 | `sync-semver-tags`| No    | If you want to keep the major and minor versions git tags synced to the latest appropriate commit <br /> (_Default: `false`_)                                                                  |
+| `notify-on-the-issue`| No       | An optional flag to enable an automatic comment on all issues linked to the release so that people following those issues get notified of the fix being released. <br /> (_Default: `true`_)                                                                  |
```

Comment:
reflect name change, plus reword to avoid implying that an issue is a bug that needs to be fixed. it can be a feature or anything else

Reply:
Updated copy:
```An optional flag to enable an automatic comment on all issues linked to each release so that people following those issues get notified of the code being released.```

Would this work or would you have an alternative suggestion?

Reply:
this looks good